socorro/external/postgresql/version_string.py

```diff
@@ -33,6 +34,7 @@ def get(self, **kwargs):
             WHERE pv.product_name = %(product)s
             AND pv.release_version = %(version)s
             AND pvb.build_id = %(build_id)s
+            AND pv.build_type = %(release_channel)s
```

Comment:
That table has "build_type" and "build_type"enum" columns. We want the "build_type" one because the other appears to have been deprecated.

This doesn't matter a whole lot because I'm rewriting all that stuff soon.